browser/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../base/.eslintrc.json",
+  "globals": {
+    "window": false,
```

Comment:
I prefer `browser`. I don't see anyone having problems then you just set the browser `env`.